demo/public/widget.html

```diff
@@ -56,11 +56,14 @@
           },
           {
             provider: 'microsoft.com',
-            providerName: 'Microsoft',
-            buttonColor: '#2F2F2F',
-            iconUrl: 'https://docs.microsoft.com/en-us/azure/active-directory/develop/media/howto-add-branding-in-azure-ad-apps/ms-symbollockup_mssymbol_19.png',
             loginHintKey: 'login_hint'
           },
+          {
+            provider: 'yahoo.com',
+          },
+          {
+            provider: 'apple.com',
```

Comment:
Does Yahoo and Apple actual work in the demo app? If not we should not add them as they will be confusing to external developers testing this.

Reply:
removed them for now.